packages/big-design/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { uniqueId } from '../../utils';
+
+import { TableContext } from './context';
+import { StyledTable, StyledTableFigure } from './styled';
+import { TableActions, TableActionsProps } from './Actions/Actions';
```

Comment:
Can we create `index.ts` files for these imports?

Reply:
Is there a reason why we haven't been doing that for `Flex` and `Grid` static members then?